src/pages/docs/getting-started/addcatalogitem.js

```diff
@@ -0,0 +1,332 @@
+import classnames from 'classnames';
+import { Popover, Transition } from '@headlessui/react';
+import Sidebar from 'components/doc/Sidebar';
+import isEmpty from 'lodash/isEmpty';
+import { graphql } from 'gatsby';
+import Prism from 'prismjs';
+
+import DocsHeader from 'components/SitewideHeader/DocsHeader';
+
+const Chip = ({ label, isActive }) => {
+  const defaultClasses =
+    'inline-flex items-center px-3 py-0.5 rounded-full text-sm font-medium mr-2';
+
+  let colorClasses;
+  if (isActive) colorClasses = 'bg-primary-100 text-primary-800';
+  if (!isActive) colorClasses = 'text-gray-800 border-2';
+
+  return <span className={classnames('capitalize', defaultClasses, colorClasses)}>{label}</span>;
+};
+
+const ShowMore = ({ item, itemName, onClick, sources }) => {
+  return (
+    <button onClick={() => onClick(item, sources)}>
+      <Chip label={itemName} />
+    </button>
+  );
+};
+
+const DefaultChip = ({ displayedChips, onClick, typeFilter, sources }) => {
+  const defaultClasses =
+    'inline-flex items-center px-3 py-0.5 rounded-full text-sm font-medium mr-2';
+
+  if (isEmpty(displayedChips)) return null;
+
+  return (
+    <>
+      <Popover className="relative">
+        {() => (
+          <>
+            <Popover.Button className={classnames(defaultClasses, 'text-gray-800 border-2')}>
+              <span>Show more</span>
+            </Popover.Button>
+
+            <Transition
+              as={Fragment}
+              enter="transition ease-out duration-200"
+              enterFrom="opacity-0 translate-y-1"
+              enterTo="opacity-100 translate-y-0"
+              leave="transition ease-in duration-150"
+              leaveFrom="opacity-100 translate-y-0"
+              leaveTo="opacity-0 translate-y-1"
+            >
+              <Popover.Panel className="absolute z-50 left-1/2 transform -translate-x-1/2 mt-3 px-2 w-screen sm:max-w-xs sm:px-0">
+                <div className="rounded-lg shadow-lg ring-1 ring-black ring-opacity-5 overflow-hidden">
+                  <div className="relative grid gap-3 bg-white px-5 py-6 sm:gap-4 sm:p-4">
+                    {displayedChips.map((item) => (
+                      <ShowMore
+                        item={item}
+                        itemName={item.frontmatter.humanName}
+                        sources={sources}
+                        key={item.name}
+                        typeFilter={typeFilter}
+                        onClick={onClick}
+                      />
+                    ))}
+                  </div>
+                </div>
+              </Popover.Panel>
+            </Transition>
+          </>
+        )}
+      </Popover>
+    </>
+  );
+};
+
+const DefaultKindChip = ({ displayedChips, onClick, typeFilter, sources }) => {
+  const defaultClasses =
+    'inline-flex items-center px-3 py-0.5 rounded-full text-sm font-medium mr-2';
+
+  if (isEmpty(displayedChips)) return null;
+
+  return (
```

Comment:
Were you able to confirm that this page does not show up in seach?

Reply:
Page doesn't show but docs I have added (for testing purposes) do get indexed as md files so they do show up. Maybe we could wait with merging until we add relevant docs?

Reply:
And of course see if we want design changed 😄